experiments/dispersy/allchannel_1000.scenario

```diff
@@ -1,6 +1,8 @@
-@0:0 set_master_member 3081a7301006072a8648ce3d020106052b81040027038192000403cbbfd2dfb67a7db66c88988df56f93fa6e7f982f9a6a0fa8898492c8b8cae23e10b159ace60b7047012082a5aa4c6e221d7e58107bb550436d57e046c11ab4f51f0ab18fa8f58d0346cc12d1cc2b61fc86fe5ed192309152e11e3f02489e30c7c971dd989e1ce5030ea0fb77d5220a92cceb567cbc94bc39ba246a42e215b55e9315b543ddeff0209e916f77c0d747
-@0:0 start_dispersy
-@0:1 online {1-980,990-1000}
+#module gumby.modules.dispersy_module.DispersyModule
```

Comment:
The thought had occured to me too. The `$` is no good, since we do environment variable expansion based on that. I used the `#` because I expect more people to associate it with preprocessor directives. On the one hand its easy since we only have 1 symbol that does something magic, the less we have of those the better imo. But yes it does impact/confuse comments. Also I found I don't like the # and m next to oneanother. Too many vertical-ish stripes. ~module ?module &module %module all look good to me.